styles/styles.css

```diff
@@ -0,0 +1,138 @@
+@import url('https://fonts.googleapis.com/css?family=Open+Sans|Ubuntu');
+
+h1, h2, h3, h4, h5, h6 {
+  font-family: Ubuntu, Helvetica, sans-serif;
+}
+
+a {
+  color: #0A1612;
+  text-decoration: none;
+  border-bottom: dotted;
+  border-width: thin;
+  border-color: #f7ce3e
+}
+a:hover{
+  /*color: #f7ce3e;*/
+  background-color: #f0f0f0;
+  border: none;
+}
+
+p, li {
+  font-family: 'Open Sans', Helvetica, sans-serif;
+  color: #0A1612;
+  /*text-decoration: none;*/
+}
+
+header {
+  position: absolute;
+  top: 0;
+  left: 0;
+  width: 100%;
+  padding: 0 3% 1% 0;
+  background-color: #f0f0f0;
+  color: #1a2930;
+}
+
+header h1 {
+  padding-left: 6%;
+}
+
+.hello, .disclaim {
+  display: inline-block;
+  width: 40%;
+  vertical-align: top;
+}
+
+.nav, .contact {
+  clear: both;
+  width: 50%;
+  display: inline-block;
+  text-align: right;
+}
+
+.nav li {
+  list-style-type: none;
+  padding-left: 10%;
+  display: inline-block;
+  color: #f0f0f0;
+  margin-top: 10px;
+}
+
+main {
+  clear: both;
+  padding: 10% 0;
+  margin: 0 6%;
+}
+
+section.welcome, section.about, section.all-interests {
+  width: 60%;
+  padding-left: 2%;
+}
+
+article.project {
+  min-width: 15%;
+  border-style: solid;
+  padding: 0 2% 0 2%;
+  margin: 4% 0;
+  vertical-align: top;
+}
+
+article.project p:first-of-type {
```

Comment:
While this is a creative use of the `first-of-type` pseudoclass, I wonder if it's really necessary in this case. What else could you use instead of a `<p>` tag as headers for your sections?